ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -62,6 +62,22 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-sts-shared-vpc-critical-high-f3
+  cron: 22 15 1,4,7,10,13,16,19,22,25,28 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-sv
```

Comment:
I didn't see this cluster profile in the profiles file. Is it merged?

Reply:
It's under reviewing, https://github.com/terraform-redhat/terraform-provider-rhcs/pull/569/files#diff-1afc179e83a78ce9bcfc4e2868861c93c2f089d81345e0c7a0d4944d80d93a40R67